cmd/wallet-web/tailwind.js

```diff
@@ -406,7 +408,82 @@ module.exports = {
         },
       };
 
-      addComponents({ ...buttons, ...inputs, ...links, ...toastNotifications });
+      const skeletons = {
+        '.skeleton': {
+          backgroundColor: theme('colors.neutrals.whiteLilac'),
+          borderRadius: theme('spacing.3'),
+          display: 'flex',
+          paddingLeft: theme('spacing.4'),
+          paddingTop: theme('spacing.4'),
+          paddingRight: theme('spacing.14'),
+          paddingBottom: theme('spacing.4'),
+          width: 342,
+          height: 80,
+          flexDirection: 'row',
+          [`@media (min-width: ${theme('screens.md')})`]: {
+            paddingLeft: theme('spacing.6'),
+            paddingTop: theme('spacing.5'),
+            paddingRight: theme('spacing.6'),
+            paddingBottom: theme('spacing.6'),
+            width: 256,
+            height: 160,
+            flexDirection: 'column',
+          },
+        },
+        '.skeleton-photo': {
+          backgroundColor: theme('colors.neutrals.selago'),
+          borderRadius: 9999,
+          width: 48,
```

Comment:
We can use value from the theme spacing here

Reply:
updated